packages/paste-core/components/card/src/Card.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, safelySpreadBoxProps } from "@twilio-paste/box";
+import type { BoxProps } from "@twilio-paste/box";
+import type { PaddingProps } from "@twilio-paste/style-props";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import * as React from "react";
+
+/** element identifier from BoxProps for customization */
+export interface CardProps extends HTMLPasteProps<"article">, PaddingProps, Pick<BoxProps, "variant"> {
```

Comment:
issue: I think variant from box is a mistake here, it doesn't do anything. I dunno why we added it either.

Reply:
yeah I was wondering about that